src/Service/Nph/NphOrderService.php

```diff
@@ -597,6 +597,34 @@ public function saveSamplesModification(array $formData, string $type, NphOrder
         return $status;
     }
 
+    public function updateSampleModificationBulk(array $formData, string $type): bool
+    {
+        $status = true;
+        foreach ($formData as $sampleId => $checked) {
+            if (isset($checked) && $checked !== null && is_bool($checked) && $checked) {
```

Comment:
This could be simplified to just $checked === true right?

Reply:
Yes you are correct.